shared/api/models/__tests__/DateModel.spec.ts

```diff
@@ -141,8 +141,8 @@ describe('DateModel', () => {
         recurrenceRule: rrulestr('DTSTART:20230414T050000\nRRULE:FREQ=WEEKLY;BYDAY=MO'),
       })
 
-      expect(date.startDate).toEqual(DateTime.fromISO('2023-10-09T07:00:00.000+02:00'))
-      expect(date.endDate).toEqual(DateTime.fromISO('2023-10-10T09:00:00.000+02:00'))
+      expect(date.startDate).toEqual(DateTime.fromISO('2023-10-02T07:00:00.000+02:00'))
```

Comment:
why did you change this?

Reply:
Now recurring events that are currently happening are also shown. Therefore the first recurrence is the 02.10.2023, not just the 09.10.2023 (as the mocked date is the 02.10.2023). However, I just changed the mocked date now.